src/avatar/Avatar.tsx

```diff
@@ -0,0 +1,35 @@
+import "./_avatar.scss";
+
+import React, {useState} from "react";
+import classNames from "classnames";
+
+export interface AvatarProps {
+  alt: string;
+  placeholderSrc: string;
+  size?: {
+    width: string;
+    height: string;
+  };
+  src?: string | null;
+  customClassName?: string;
+}
+
+function Avatar({alt, size, src, customClassName, placeholderSrc}: AvatarProps) {
+  const [shouldDisplayPlaceholder, setPlaceholderVisibility] = useState(true);
```

Comment:
I don't understand the need for this state. After making the `placeholderSrc` optional, we can just check if the value is truthy and render `img` with `placeholderSrc` if `src` is falsy. If both of them are falsy, then we shouldn't render any `img` tag I think.

Actually, I am not really sure why we are adding a separate `placeholderSrc` prop. As a user of this component, I could just provide `src` appropriately if I need to render a placeholder image.